parcels/scripts/get_examples.py

```diff
@@ -94,8 +94,9 @@ def main(target_path=None):
         target_path = args.target_path
 
     if os.path.exists(target_path):
-        print("Error: {} already exists.".format(target_path))
-        return
+        answer = raw_input("Warning: {} already exists. Continue [y/n]?".format(target_path)).lower()
```

Comment:
I'd suggest: "Continue and overwrite existing example files [y/N]?!"

To make sure a user knows that existing files may be lost and that the default answer (Just hitting Return.) amounts to: "Don't continue."

Reply:
OK, I will change